plugins/woocommerce/src/Admin/Features/OnboardingTasks/TaskList.php

```diff
@@ -341,7 +354,7 @@ public function possibly_track_completion() {
 
 		$completed_lists   = get_option( self::COMPLETED_OPTION, array() );
 		$completed_lists[] = $this->get_list_id();
-		update_option( self::COMPLETED_OPTION, $completed_lists );
+		update_option( self::COMPLETED_OPTION, $completed_lists, true );
```

Comment:
Enable `autoload` for the option since we're using it on every WC admin page load.